docs/administration-guide/databases/mongodb.md

```diff
@@ -0,0 +1,55 @@
+# Working with MongoDB in Metabase
+
+
+This guide covers:
+
+ - What's new in `0.32.9`
+ - General connectivity concerns.
+ - Connecting to a MongoDB Atlas cluster.
+
+## What's new in `0.32.9`
+
+ - In `0.32.9`, Metabase adds support/fixes for certain MongoDB connection configurations, such as:
+   1. When connecting to an Atlas cluster, connect using `DNS SRV`, which is the recommended and default method for newer clusters.  
```

Comment:
I don't think that docs are the right place for this type of "here's what's new" content; we usually try and do that in release notes. I think that the section below where you cover how to connect to Atlas clusters is sufficient for calling these changes out. I'd cut the "What's new in 0.32.9" bullet point and this paragraph because they won't age well.